src/mango/test/12-use-correct-index-test.py

```diff
@@ -54,36 +54,41 @@ def setUp(self):
         self.db.save_docs(copy.deepcopy(DOCS))
 
     def test_choose_index_with_one_field_in_index(self):
-        self.db.create_index(["name", "age", "user_id"], ddoc="aaa")
-        self.db.create_index(["name"], ddoc="zzz")
+        self.db.create_index(["name", "age", "user_id"], ddoc="aaa", wait_for_built_index=False)
```

Comment:
A lot of this code churn would be cleaned up if we flipped the default on the `wait_for_built_index` and only not wait when we're allowing indexes to build in parallel.

Reply:
From the mailing list discussion I think we going to keep the wait_for_build_index design

Reply:
Right, I was just suggesting that to clean up the tests it might help to flip the default wait from `True` to `False` so that there are fewer changed lines in the diff. Not a big deal.